README.rst

```diff
@@ -287,18 +246,14 @@ Use this installation mode if you are contributing to NeMo.
     cd NeMo
     ./reinstall.sh
 
-If you only want the toolkit without additional conda-based dependencies, you may replace ``reinstall.sh``
-with ``pip install -e .`` when your PWD is the root of the NeMo repository.
+If you only want the toolkit without the additional Conda-based dependencies, you can replace ``reinstall.sh`` with ``pip install -e .`` when your PWD is the root of the NeMo repository.
 
-Mac computers with Apple silicon
+Mac Computers with Apple Silicon
```

Comment:
Sidenote: This is ONLY applicable to ASR domain specifically. Maybe we should mention this somewhere

Reply:
Added a note

Reply:
OK